File: src/main/kotlin/net/wavem/uvc/rms/gateway/path/domain/Path.kt
package net.wavem.uvc.rms.gateway.path.domain

import net.wavem.uvc.rms.common.domain.header.Header
import net.wavem.uvc.rms.gateway.path.domain.job.info.PathJobInfo
import net.wavem.uvc.rms.gateway.path.domain.job.kind.PathJobKind
import net.wavem.uvc.rms.gateway.path.domain.job.path.PathJobPath

data class Path(
    val header: Header?,
    val jobInfo: PathJobInfo?,
    val jobPath: PathJobPath?,
    val jobKind: PathJobKind?
) {
}

File: src/rms/request/path/domain/path.py

class Path():
    def __init__(
        self,
        header: dict = None,
        jobInfo: dict = None,
        jobPath: dict = None
    ) -> None:
        self.header: dict = header
        self.jobInfo: dict = jobInfo
        self.jobPath: dict = jobPath
        
        
__all__ = ['path']